src/middlewared/middlewared/etc_files/krb5.keytab.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import base64
+
+from freenasUI.common.pipesubr import pipeopen
```

Comment:
Can we please try to not use freenasUI in middlewared unless strictly necessary?